pillar/redash.sls

```diff
@@ -6,7 +20,34 @@ apache:
       servername: redash.open-contracting.org
       context:
         proxypass: http://localhost:9090/
+
+postgres:
+  version: 13
+  # Redash connections to Postgres are not sent over the localhost connection.
```

Comment:
Can you remind me why this is?

Reply:
If I remember correctly, it's because Docker uses the external IP.

Reply:
Correct - Docker is connecting to Postgres in a strange way.

Containers are running in their own network on the `172.17.0.0/16` range (at least by default).
The Redash containers are not aware of any network outside of itself, additionally "localhost" is the container not the server.